### PR TITLE
Add Monokai Pro Machine, Ristretto, Octagon, Spectrum

### PR DESCRIPTION
**Pull request by minikN**
Hello,

I've added the missing filters:
- Octagon
- Ristretto
- Machine
- Spectrum

# Spectrum
image

# Ristretto
image

# Machine
image

# Octagon
image

### CODE REVIEW

**Comment by minikN** [comment]
Reference: #532

File: themes/doom-monokai-octagon-theme.el [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+  :group 'doom-monokai-octagon-theme
+  :type '(choice integer boolean))
+
+;;
+(def-doom-theme doom-monokai-octagon
+  "A dark, vibrant theme inspired by Textmate's Monokai."
+
+  ;; name        gui       256       16
+  ((bg         '("#282a3a" nil       nil            ))
+   (bg-alt     '("#161821" nil       nil            ))
```
Comment (ema2159) [inline themes/doom-monokai-octagon-theme.el:32]: Change bg-alt to #1E1F2B

File: themes/doom-monokai-ristretto-theme.el [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+  :group 'doom-monokai-ristretto-theme
+  :type '(choice integer boolean))
+
+;;
+(def-doom-theme doom-monokai-ristretto
+  "A dark, vibrant theme inspired by Textmate's Monokai."
+
+  ;; name        gui       256       16
+  ((bg         '("#2c2525" nil       nil            ))
+   (bg-alt     '("#191515" nil       nil            ))
```
Comment (ema2159) [inline themes/doom-monokai-ristretto-theme.el:32]: Change bg-alt to #201C1C

File: themes/doom-monokai-machine-theme.el [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+  :group 'doom-monokai-machine-theme
+  :type '(choice integer boolean))
+
+;;
+(def-doom-theme doom-monokai-machine
+  "A dark, vibrant theme inspired by Textmate's Monokai."
+
+  ;; name        gui       256       16
+  ((bg         '("#273136" nil       nil            ))
+   (bg-alt     '("#161b1e" nil       nil            ))
```
Comment (ema2159) [inline themes/doom-monokai-machine-theme.el:32]: Change bg-alt to #1E2528

**Review by ema2159 — CHANGES_REQUESTED** [review]

**Review by ema2159 — APPROVED** [review]

**Comment by ema2159** [comment]
Excellent job man. Sorry for the delay!

**Comment by ema2159** [comment]
One very last thing. If I'm not wrong, if I merge this as it is right now, it will delete the current Monokai pro theme. In order to prevent this I think the only thing to be done is to create again the doom-monokai-pro.el file with the contents of the current implementation. I was going to do so myself but it seems I can't so could you please do this as soon as you have time?

**Pull request merged**